helpers.js

```diff
@@ -492,6 +492,16 @@ const getArmletClient = (ethAddress, password, clientToolName = 'truffle') => {
  * @param {Object} config - truffle configuration object.
  */
 async function analyze(config) {
+
+    try {
+      const projectConfig = require([config.working_directory,
+				       'truffle-security.json'].join ('/'));
+      config = Object.assign(config, projectConfig);
```

Comment:
My understanding is that this will overwrite passed in arguments with the configuration file. I might be misinterpreting this though.

The configuration priority should probably be first command line arguments, then fall back to `truffle-security.json`, then fall back to the defaults.

Reply:
Nice catch. Yes you are totally right that this was in need of changing. I've included a comment in the update that command line args will override the project level config :+1:  

Since at this point truffle has taken care of merging the command line args with the default case... I'm not messing with that for now, but let me know if you think more should be done here.